src/project-management/AppMetadata.ts

```diff
@@ -0,0 +1,41 @@
+/*!
```

Comment:
File should be renamed to match the current convention in use. app-metadata.ts or similar.